notebooks/125-torchvision-zoo-to-openvino/125-convnext-classification.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<p>looks like white space missed  use<a href="https://pytorch.org/vision/stable/models.html#using-the-pre-trained-models" target="_blank">preprocessing transforms from the model wight</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
fixed